vendor/cran.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'cran', title: 'CRAN/METACRAN' });
+module.exports = t;
+
+t.create('version')
+  .get('/v/devtools.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('cran'),
+    value: Joi.string().regex(/^v\d+\.\d+\.\d+$/)
+  }));
+
+t.create('specified license')
+  .get('/l/devtools.json')
+  .expectJSON({ name: 'license', value: 'GPL (>= 2)' });
+
+t.create('unknown package')
+  .get('/l/some-bogus-package.json')
```

Comment:
Whenever people make typos while setting up their badges, this is the type of request Shields will send from the production server, so I don't see a problem issuing it from the test. Mocking it would defeat the purpose, which is in part to find out if service has changed its API contract. For that we need to issue a request to the real server and see how it responds.